### ISSUE TITLE
Different icon for taxis other than passenger cars

### ISSUE THREAD

**Issue by govvin** [gravitystorm/openstreetmap-carto#3621]
As suggested in #879 , I'm opening a separate ticket.

Kindly re-consider a new different icon/pictogram for this feature. The western taxi cab is not appropriate elsewhere, where taxi services are also offered - in addition to regular taxis, other forms: tuktuks (auto rickshaws), pedicabs (cycle rickshaws), and moto-taxis.

I've been looking at icons/pictograms at the Noun project, and found interesting ones, but I don't I found anything "universal" enough.

The earlier suggestion by daganzdaanda in #879 to use "taxi" looks like a good starting point, especially the first one - short and sweet:

### Expected behavior
a different icon, for example, something like one of these, for starters? 

![](https://cloud.githubusercontent.com/assets/7693297/6158745/d7deefdc-b24d-11e4-829b-71d36dcf2c63.png)

**Comment by kocio-pl** [comment]
I don't like any of them, and at 14px (our icon matrix) it might look bad, I guess.

Car is not used for anything else, so there's no problem recognizing it, and it's better to avoid letters to keep symbols more universal.

**Comment by sommerluk** [comment]
We avoid text as icons. Not everyone can read Latin script (though for sure it’s the most extended script in the world). And not everyone understands the word “taxi” (though in many languages the word is similar). Text should be avoided. A car symbols seems to general for me (will likely be confusing).

I would suggest to take the alternating rows of squares as a starting point. An example would be https://github.com/mapsme/omim/blob/master/data/styles/clear/style-night/symbols/taxi-m.svg which looks like this:
![screenshot 1](https://user-images.githubusercontent.com/6830724/50683460-11735800-100a-11e9-85f3-7a465e6a3125.png)
Not quite our icon style, and as every symbol not 100% universal, but I think the alternating rows of squares could be a good starting point. See also [this image](https://pixabay.com/de/ny-new-york-taxi-innenstadt-2866358/).

**Comment by govvin** [comment]
@sommerluk , I did find that symbology interesting.  I also found [this](https://history.stackexchange.com/questions/4490/when-did-the-taxi-become-associated-with-the-checkered-pattern), which explains how the checker symbol got affiliated with taxi cabs.

I found similar designs from [Noun](https://thenounproject.com/term/taxi/), while reviewing for alternative taxi symbology:

![image](https://user-images.githubusercontent.com/5821688/50685573-b0bd3d00-1054-11e9-81d0-bf3f298b675e.png)

![image](https://user-images.githubusercontent.com/5821688/50685479-56bc7780-1054-11e9-9ab7-415bdc34ab15.png)

![image](https://user-images.githubusercontent.com/5821688/50685536-89667000-1054-11e9-9501-28cff60da6c1.png)

![image](https://user-images.githubusercontent.com/5821688/50685566-a7cc6b80-1054-11e9-8cf7-53c36fb3c1b7.png)

![image](https://user-images.githubusercontent.com/5821688/50685581-b9ae0e80-1054-11e9-92f9-fcd43e26d7a8.png)

![image](https://user-images.githubusercontent.com/5821688/50685594-c3377680-1054-11e9-841d-3eb98ce802c6.png)

![image](https://user-images.githubusercontent.com/5821688/50685675-17daf180-1055-11e9-8db1-0a8fa46da666.png)


(All images we're found from the Noun project, and just pasting here for convenience and sake of discussion. )

**Comment by jeisenbe** [comment]
We try to avoid using words and letters in icons.

But it would be great if we could use separate icons for motorcycle
"taxis" and pedicabs.

Here in Indonesia, we have a choice of hired cars (taxi), auto
rickshaws (Bajai), pedicabs (becak ), and motorcycle taxis (ojek).
These are all considered different forms of transport.

Perhaps we need separate icons for motorcycle “taxis” and pedicabs.
"Ojek" (motorcycle "taxi") stands often have a shelter and a sign with
a name, and serve as a bit of a local landmark in Indonesia; then is
one in every neighborhood, so I suspect there are at least 10,000 in
this country alone, and probably just as many in the rest of southeast
Asia.

According to the amenity=taxi wiki page, one can tag amenity=taxi with
motorcycle=yes and motorcar=no for a motorcycle-only "taxi" stand.
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dtaxi

However, the Indonesian tagging page
(https://wiki.openstreetmap.org/wiki/Indonesian_Tagging_Guidelines)
suggests using amenity=shelter with shelter=public_transport, which is
less specific, so very few of them are tagged as amenity=taxi. (The
local language page is the same). I only found 175 amenity=taxi with
motorcycle=yes on overpass-turbo.

The Philippines page suggests tagging pedicabs as:
amenity=taxi
cycle_rickshaw=yes
motorcar=no
motorcycle=no

Yikes, who is going to do that without a preset?

So while it would be nice to render different types of hired vehicles,
it looks like the tagging needs to be decided first.

-Joseph

**Comment by polarbearing** [comment]
I'm not convinced that we should consider other vehicle types than cars for amenity=taxi, unless there is evidence that it is massively used for bicycles. 

Thus we could use the shop=car symbol in transport colour, or have a small sign on top, already proposed in the other thread:
<img src="https://cloud.githubusercontent.com/assets/3531092/7732767/71c4e18c-ff2a-11e4-97fa-f386a2645e57.png">

**Comment by govvin** [comment]
> I'm not convinced that we should consider other vehicle types than cars for amenity=taxi, unless there is evidence that it is massively used for bicycles.
> 
> Thus we could use the shop=car symbol in transport colour, or have a small sign on top, already proposed in the other thread:
> ![](https://cloud.githubusercontent.com/assets/3531092/7732767/71c4e18c-ff2a-11e4-97fa-f386a2645e57.png)

In many towns in the Philippines, including small cities,  where taxi cabs are **not** available, pedicabs, rickshaws, and moto-taxis serve the same purpose as a taxi cab.

**Comment by polarbearing** [comment]
I got confused with the ref to  #879, and understood this thread title as a request to render taxis in general. #1562 was the PR that did that. So this issue is about changing the icon, that should be renamed. 

Still then, using a car as taxi is so common that it should _**not**_ be changed. If specific cases as rickshaws need be considered, than a separate icon should be used.

**Comment by kocio-pl** [comment]
I've found documented tag sets for taxis other than car:

https://wiki.openstreetmap.org/wiki/Philippines/Mapping_conventions#Commuter_Transportation_.2F_Auto_Shops_.26_Services

**Comment by wilmaed** [comment]
>  I did find that symbology interesting. I also found [this](https://history.stackexchange.com/questions/4490/when-did-the-taxi-become-associated-with-the-checkered-pattern), which explains how the checker symbol got affiliated with taxi cabs.
> 

Is anyone using this symbol outside of the usa ?
I have never seen it ...

**Comment by sommerluk** [comment]
> Still then, using a car as taxi is so common that it should not be changed.

Well, a car could also symbolise a car repair or rent. It's not specific to taxis.

**Comment by kocio-pl** [comment]
Right - yet not blue one, which is for transport (not shops or other services).

Anyway, it looks to me that amenity=taxi needs discussion to make it clear when it is meant for cars and how rickshaws should be tagged before we try to change rendering.

**Comment by Rainer-o** [comment]
I woul abstain from the chess squares. That may be used in some countries, but I doubt anybody in all the countries that don't use that, will associate it with taxi (honestly I was thinking of the cars on the airport runways, but never of a taxi). The word Taxi is known, so I would prefer to have something like the sign in the bottom of govvin that shows perspectively the roof sign of a taxi, but fill it with the letters TAXI.

**Comment by polarbearing** [comment]
The issue here is not about finding a symbol for passenger-car taxi, we have that already. 
As apparently more people getting confused:
* could a maintainer please change the title to "Different icon for taxis other than passenger cars" or so.
* I propose to close the issue for now, because there are less than 300 such cases, with a poorly constructed tagging scheme. The scheme itself is currently discussed on the tagging list.

**Comment by kocio-pl** [comment]
I'm closing it now, since tagging is unclear at the moment. Whet it will be ready and some usage will be there, we can reopen it, however probably new icon should be properly designed anyway.
